.github/workflows/ci.yml

```diff
@@ -129,12 +93,17 @@ jobs:
         env:
           CASTOR_BIN: ${{ github.workspace }}/${{ matrix.castor.bin }}
 
-  phar:
-    name: Ensure phar is OK
+  artefacts:
+    name: Ensure artefacts are OK
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout
+        uses: actions/checkout@v4
 
-      - name: Build all the phars
-        uses: ./.github/actions/phar
+      - uses: ./.github/actions/install
+        with:
+          composer-flags: "--no-dev"
+      - uses: ./.github/actions/cache
+      - uses: ./.github/actions/phar
+      - uses: ./.github/actions/static
```

Comment:
Looks like this line break the CI. Should we remove it? (it's not commited atm)

Reply:
no, we need it :) even if we don't publish it yet we must ensure it's okay